tests/testthat/test-r-lib.R

```diff
@@ -0,0 +1,28 @@
+context("r-lib")
```

Comment:
If we *don't* put lifecycle in Suggests, all 3 of these tests need to mock `usethis:::is_installed` the way we do for packages like learnr and RcppArmadillo:

https://github.com/r-lib/usethis/blob/187bad13a10c8a85e517205936d45338978b1cb6/tests/testthat/test-use-tutorial.R#L14

https://github.com/r-lib/usethis/blob/187bad13a10c8a85e517205936d45338978b1cb6/tests/testthat/test-use-rcpp.R#L24